zerver/openapi/zulip.yaml

```diff
@@ -1382,37 +1382,34 @@ paths:
                                     - delete
                                 streams:
                                   type: array
+                                  deprecated: true
                                   description: |
-                                    Array of objects, each containing
-                                    details about a channel that was deleted.
+                                    Array of objects, each containing stream_id of the channel
+                                    that was deleted.
+
+                                    **Deprecated** and will be removed in a future release.
                                   items:
-                                    $ref: "#/components/schemas/BasicChannel"
+                                    type: object
+                                    properties:
+                                      stream_id:
+                                        type: integer
+                                        description: |
+                                          Stream Id associated with the event deletion
+                                    additionalProperties: false
+                                stream_ids:
+                                  type: array
+                                  description: |
+                                    Array of integers, each containing stream_id
```

Comment:
This can be better phrased as `Array containing IDs of the channels that were deleted`.